src/style.css

```diff
@@ -1 +1,53 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  margin: 0;
+  padding: 0 50px;
+  font-weight: 500;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-top: 0;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav__link {
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  text-decoration: none;
+  padding: 22px 20px;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 56px;
+  width: 37px;
```

Comment:
Same here. If you replace it with `left: 0` and `right: 0` it will work with any width.